### ISSUE TITLE
Note embedding does not work in 1.8.0-10b

### ISSUE THREAD

**Issue by tobemo** [KosmosisDire/obsidian-webpage-export#349]
**Describe the bug**
I discovered this bug when trying to embed notes inside callouts. At first I tried it on 1.7.3 but there it would:
- not work with notes that were collapsed by default;
- work with notes that were expanded by default;
- not work lower down the page (so maybe the first 3 would work and later ones wouldn't);
- be a hit or miss with regular embeddings (so not inside a callout), especially if using nested embedding.

Then I switched to the beta version 1.8.0-10b (which has a nicer user experience btw👍).
I also disabled the Minimal theme which gave some issues on v1.7.3 so I think I'm on quite a clean install.

Here embedding notes using the `![[Name of note]]` syntax inside of a callout does not export properly to HTML. Only the title of the embedded note renders. I have tried this for all 3 formats (Online Web Server, HTML documents and Raw HTML documents) but am mostly interested in a single Raw HTML file. When using more website-like version I disabled most/all page features (Theme toggle, Document outline, navigation tree).

**Operating System**
Windows 10
Tried on:
- Chrome
- Edge
- Firefox

**To Reproduce**
1. Create a file called 'Secondary' with the content:
```
Text to embed.
```
2. Create a file called 'Main' with the content:
```
> [!info] Reference
> Text that is not embedded works great.

>[!info] Default
>![[Secondary]]

>[!info]- Collapsed
>![[Secondary]]

>[!info]+ Expanded
>![[Secondary]]

![[Secondary]]
```
3. Export this file 'Main' as a Raw HTML Document.

**Screenshots**
*On web:*
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/36672287/d5384e82-1214-4593-a756-09c55622993d)
*In Obsidian:*
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/36672287/3f26f976-8508-4698-a3d7-4f620de4a251)

### IMPORTANT: Error Log
There are no errors.
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/36672287/55f7e96f-36c0-4bda-90ff-d18ea583de98)

Thank you for this repo, I appreciate the effort.

**Comment by KosmosisDire** [comment]
Thank you for this report, I'll look into fixing this

**Comment by KosmosisDire** [comment]
I was able to reproduce and fix this issue, the fix will be out in the next update

**Comment by KosmosisDire** [comment]
Can you verify this is fixed in 1.8.0-11b?

**Comment by tobemo** [comment]
Thank you for the effort!
